packages/aws-cdk-lib/core/test/staging.test.ts

```diff
@@ -1260,6 +1260,53 @@ describe('staging', () => {
       },
     })).toThrow(/Bundling output directory is expected to include only a single archive file when `output` is set to `ARCHIVED`/);
   });
+
+  test('bundling that produces a single non archive file with FILE', () => {
+    // GIVEN
+    const app = new App({ context: { [cxapi.NEW_STYLE_STACK_SYNTHESIS_CONTEXT]: false } });
+    const stack = new Stack(app, 'stack');
+    const directory = path.join(__dirname, 'fs', 'fixtures', 'test1');
+
+    // WHEN
+    const staging =new AssetStaging(stack, 'Asset', {
+      sourcePath: directory,
+      bundling: {
+        image: DockerImage.fromRegistry('alpine'),
+        command: [DockerStubCommand.SUCCESS],
+        outputType: BundlingOutput.FILE,
+      },
+    });
+
+    // THEN
+    const assembly = app.synth();
+    expect(fs.readdirSync(assembly.directory)).toEqual([
+      'asset.f5160e037189d688a526fefb9c0ef62e8638115ba7ccc90011e6e0aecb05f39d',
```

Comment:
It looks like it is still staging both the directory and the file, can you check on that?